src/lib/y2storage/bcache.rb

```diff
@@ -84,14 +108,33 @@ class Bcache < Partitionable
     #   @return [String] full path to new bcache device like "/dev/bcache3"
     storage_class_forward :find_free_name
 
+    # Whether the Bcache is flash-only
+    #
+    # @return [Boolean]
+    def flash_only?
+      type == BcacheType::FLASH_ONLY
```

Comment:
NP: a possible (nicer?) alternative: `type.is?(:flash_only)`

Reply:
Sure, much nicer ;). Thanks.